docker/build_jakartaeetck.sh

```diff
@@ -95,8 +95,8 @@ unzip -o latest-glassfish.zip
 ls -l $GF_HOME/glassfish5/glassfish/
 
 #temporary fix to get build passing until we have glassfish with new api jars
-wget --progress=bar:force --no-cache https://repo1.maven.org/maven2/jakarta/platform/jakarta.jakartaee-api/9.0.0-RC1/jakarta.jakartaee-api-9.0.0-RC1.jar 
-mv jakarta.jakartaee-api-9.0.0-RC1.jar $GF_HOME/glassfish5/glassfish/modules
+wget --progress=bar:force --no-cache https://repo1.maven.org/maven2/jakarta/platform/jakarta.jakartaee-api/9.0.0-RC1/jakarta.jakartaee-api-9.0.0-RC1.jar -O ${BASEDIR}/lib/jakartaee-api.jar
+cp ${BASEDIR}/lib/jakartaee-api.jar $GF_HOME/glassfish5/glassfish/modules
```

Comment:
the lib folder is a folder that is packed to the final CTS/TCK bundle, and the Jakarta EE API should not be part of the lib folder of the CTS bundle. This why it is moved to the GF, not copied.

Reply:
this is now addressed, by downloading directly to $GF_HOME/glassfish5/glassfish/modules/jakartaee-api.jar.  Resolving this issue.